lib/particular-calendars/philippines.ts

```diff
@@ -50,7 +50,7 @@ export class Philippines extends CalendarDef {
     },
 
     ezequiel_moreno_bishop: {
-      precedence: Precedences.OptionalMemorial_12,
+      precedence: Precedences.ProperMemorial_11b, // src: https://recoletos.ph/2024/02/01/cbcp-elevates-st-ezekiel-moreno-memorial-as-obligatory/; since: 29 Jan 2024; until then it was an optional memorial
```

Comment:
proposal: 
```suggestion
      /*
        src: https://recoletos.ph/2024/02/01/cbcp-elevates-st-ezekiel-moreno-memorial-as-obligatory/;
        since 29 Jan 2024; until then it was an optional memorial
       */
      precedence: Precedences.ProperMemorial_11b,
```

Reply:
I really don’t care about the comment format here, as I consider them as notes for the fure future source and reference objects.

That said, until now, we added only `src` and `based on` comments after the particular l10n key value. Here, I have added a note of the previous rank and when it was changed.

We could also use a JSON string comment instead, however, as I have already said, it does not matter to me much … until we add actual sources and references into romcal.

Reply:
yes, i'm not debating what the content should be, precisely, just that a block comment is better for reading and formatting this long text than a line comment after the content

Reply:
Okay, I’ll accept your suggestion. :+1:

Reply:
I have changed it to a YAML within a block comment. I hope you are okay with this. :wink:

```ts
      /*
        src: https://recoletos.ph/2024/02/01/cbcp-elevates-st-ezekiel-moreno-memorial-as-obligatory/;
        old:
          rank: Precedences.OptionalMemorial_12
          since: unknown
          until: 28 Jan 2024
        new:
          rank: Precedences.ProperMemorial_11b
          since: 29 Jan 2024
       */
```

Reply:
Actually, I am thinking about changing it to this:

```ts
      /*
       * src: https://recoletos.ph/2024/02/01/cbcp-elevates-st-ezekiel-moreno-memorial-as-obligatory/;
       * old:
       *   rank: Precedences.OptionalMemorial_12
       *   since: unknown
       *   until: 28 Jan 2024
       * new:
       *   rank: Precedences.ProperMemorial_11b
       *   since: 29 Jan 2024
       */
```

Reply:
Changed.